### DISCUSSION TITLE
Github Action: Support for multiple schema files

### DISCUSSION THREAD

**Discussion by craig-day** [kamilkisiela/graphql-inspector#1531] [General] [answered]
The CLI and JS packages support passing in multiple schema files to perform the inspection on, but currently the Github action seems to only support a single schema file. 

It would be great to support multiple files in the predefined action to keep functional parity with the other usages of the tool.

**Comment by Urigo** [comment, marked as answer]
Discussed here: https://github.com/kamilkisiela/graphql-inspector/issues/1978
